apps/blockchain/test/blockchain/account/repo_test.exs

```diff
@@ -184,6 +184,32 @@ defmodule Blockchain.Account.RepoTest do
 
       assert found_code == code
     end
+
+    test "caches machine code and then always reads it from cache", %{state: state} do
+      address = <<1>>
```

Comment:
What do you think about using an address that is the same size as the ones on real accounts? I think it's something like `<<1::256>>`, right?